kboparai1/ex10/sd01.py

```diff
@@ -0,0 +1 @@
+# I'll not be using flash cards. but i will google them!
```

Comment:
Try to add a newline at the end of the file, it's both style and could cause problems in some circumstances... i forget them often too...

<img width="515" alt="screen shot 2016-01-03 at 8 41 28 pm" src="https://cloud.githubusercontent.com/assets/9515253/12083557/6e32305e-b25a-11e5-800a-ba366abb8b8d.png">